File: Servidor/Transaccional/AccesoDatos/Dao/NHibernate/DaoObjetoNHibernate.cs
using NLog;
using Trascend.Bolet.AccesoDatos.Contrato;
using Trascend.Bolet.ObjetosComunes.Entidades;

namespace Trascend.Bolet.AccesoDatos.Dao.NHibernate
{
    public class DaoObjetoNHibernate : DaoBaseNHibernate<Objeto, string>, IDaoObjeto
    {
        private static Logger logger = LogManager.GetCurrentClassLogger();
    }
}

File: src/Servidor/Transaccional/AccesoDatos_Fac/Dao/NHibernate/DaoObjetoNHibernate.vb
Imports NLog
Imports Diginsoft.Bolet.AccesoDatos.Contrato
Imports Diginsoft.Bolet.ObjetosComunes.Entidades

Namespace Dao.NHibernate
    Public Class DaoObjetoNHibernate
        Inherits DaoBaseNHibernate(Of Objeto, String)
        Implements IDaoObjeto
        Private Shared logger As Logger = LogManager.GetCurrentClassLogger()
    End Class
End Namespace